course/lesson-5-your-first-function/lesson.tres

```diff
@@ -23,7 +23,7 @@ script = ExtResource( 5 )
 content_id = "res://course/lesson-5-your-first-function/content-TtcyhcIb.tres"
 title = "Functions are named sequences of instructions"
 type = 0
-text = "Functions are [i]sequences of instructions[/i] we give a name. We call that name an [i]identifier[/i].
+text = "Functions are [i]sequences of instructions[/i] we give a name to. We call that name an [i]identifier[/i].
```

Comment:
These kinds of changes I'm not fond of because it's grammatically correct and understandable before and after. But it'll require updating the translation string in every language.

Note that we have a professional editor in the team. I trust him to make the phrasing natural and clear.